optimade/filtertransformers/mongo.py

```diff
@@ -155,10 +176,15 @@ def set_op_rhs(self, arg):
 
     def length_op_rhs(self, arg):
         # length_op_rhs: LENGTH [ OPERATOR ] value
-        # TODO: https://stackoverflow.com/questions/7811163/query-for-documents-where-array-size-is-greater-than-1
         if len(arg) == 2 or (len(arg) == 3 and arg[1] == "="):
             return {"$size": arg[-1]}
 
+        # the approach taken in post-process still cannot handle !=, so return error here
```

Comment:
```suggestion
```

Reply:
I am not completely seeing that these comments make sense now - it may be a leftover from previous run throughs, I'm guessing? If not, then please make these comments clearer, since I am not seeing you returning an error here :)

On closer inspection, I presume you mean the `raise NotImplementedError` below? I would suggest either putting this comment together with the rest under the `elif` or remove it, since the code already says that you cannot handle `!=`.